<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Chronos Screen</string>
  </property>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>App &amp; Internet Access</string>
   </property>
  </widget>
  <widget class="LineEdit" name="uiPassword">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>LineEdit::none</enum>
   </property>
   <property name="linkedValueName" stdset="0">
    <string>networkPassword</string>
   </property>
  </widget>
  <widget class="Label" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
			font-size: 17px;
			background: transparent;
		</string>
   </property>
   <property name="text">
    <string>Remote Access Password</string>
   </property>
  </widget>
  <widget class="Button" name="uiShowPassword">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>üëÅ</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::half</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiRandomisePassword">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>üé≤</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="QWidget" name="uiAllConfigOptions" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="CheckBox" name="uiEnableRemoteSSH">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Remote SSH Access</string>
    </property>
    <property name="clickMarginTop" stdset="0">
     <enum>CheckBox::none</enum>
    </property>
    <property name="linkedValueName" stdset="0">
     <string>remoteSSHAccess</string>
    </property>
   </widget>
   <widget class="Label" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
			font-size: 17px;
			background: transparent;
		</string>
    </property>
    <property name="text">
     <string>Local Area Network</string>
    </property>
   </widget>
   <widget class="Label" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Allow access to this camera from the internet at large. Set a strong Access Password! Please refer to the manual for additional details.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="CheckBox" name="uiEnableLocalSSH">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Local SSH Access</string>
    </property>
    <property name="clickMarginTop" stdset="0">
     <enum>CheckBox::none</enum>
    </property>
    <property name="linkedValueName" stdset="0">
     <string>localSSHAccess</string>
    </property>
   </widget>
   <widget class="Label" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
			font-size: 17px;
			background: transparent;
		</string>
    </property>
    <property name="text">
     <string>World Wide Web</string>
    </property>
   </widget>
   <widget class="CheckBox" name="uiEnableLocalHTTP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Local HTTP Access &amp;&amp; Web App</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="clickMarginBottom" stdset="0">
     <enum>CheckBox::none</enum>
    </property>
    <property name="linkedValueName" stdset="0">
     <string>localHTTPAccess</string>
    </property>
   </widget>
   <widget class="CheckBox" name="uiEnableRemoteHTTP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Remote HTTPS Access</string>
    </property>
    <property name="clickMarginBottom" stdset="0">
     <enum>CheckBox::none</enum>
    </property>
    <property name="linkedValueName" stdset="0">
     <string>remoteHTTPAccess</string>
    </property>
   </widget>
   <widget class="Label" name="label_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
			font-size: 17px;
			background: transparent;
		</string>
    </property>
    <property name="text">
     <string>Network Address Information</string>
    </property>
   </widget>
   <widget class="Label" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Interface</string>
    </property>
   </widget>
   <widget class="ComboBox" name="uiNetworkInterface">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="clickMarginBottom" stdset="0">
     <enum>ComboBox::half</enum>
    </property>
   </widget>
   <widget class="Label" name="uiNoNetworkConnection">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>no network connected</string>
    </property>
   </widget>
   <widget class="QWidget" name="uiNetworkInterfaceInformation" native="true">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>351</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="Button" name="uiRandomiseHTTPPort">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>üé≤</string>
     </property>
     <property name="clickMarginLeft" stdset="0">
      <enum>Button::half</enum>
     </property>
     <property name="clickMarginTop" stdset="0">
      <enum>Button::half</enum>
     </property>
    </widget>
    <widget class="SpinBox" name="uiHTTPPort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>80</number>
     </property>
     <property name="clickMarginRight" stdset="0">
      <enum>SpinBox::none</enum>
     </property>
     <property name="clickMarginTop" stdset="0">
      <enum>SpinBox::half</enum>
     </property>
     <property name="linkedValueName" stdset="0">
      <string>HTTPPort</string>
     </property>
    </widget>
    <widget class="Label" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>SSH Port</string>
     </property>
    </widget>
    <widget class="Label" name="uiAppUrl">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>66</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>http://192.168.100.100/</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="SpinBox" name="uiSSHPort">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>22</number>
     </property>
     <property name="clickMarginRight" stdset="0">
      <enum>SpinBox::none</enum>
     </property>
     <property name="linkedValueName" stdset="0">
      <string>SSHPort</string>
     </property>
    </widget>
    <widget class="Label" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>App URL</string>
     </property>
    </widget>
    <widget class="Label" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>HTTP Port</string>
     </property>
    </widget>
    <widget class="Label" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Command</string>
     </property>
    </widget>
    <widget class="Label" name="uiSSHCommand">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>176</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ssh root@192.168.100.100
    -p 22786</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="Button" name="uiRandomiseSSHPort">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>71</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>üé≤</string>
     </property>
     <property name="clickMarginLeft" stdset="0">
      <enum>Button::half</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Label" name="uiPasswordExplanation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Before enabling network control for your camera, you must enter a password to use. You can change this password at any time.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>400</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>uiAllConfigOptions</zorder>
  <zorder>headerlabel</zorder>
  <zorder>label</zorder>
  <zorder>uiPasswordExplanation</zorder>
  <zorder>uiShowPassword</zorder>
  <zorder>uiRandomisePassword</zorder>
  <zorder>uiPassword</zorder>
  <zorder>uiDone</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>CheckBox</class>
   <extends>QCheckBox</extends>
   <header>check_box</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combo_box</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>line_edit</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spin_box</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiDone</tabstop>
  <tabstop>uiPassword</tabstop>
  <tabstop>uiShowPassword</tabstop>
  <tabstop>uiRandomisePassword</tabstop>
  <tabstop>uiEnableLocalHTTP</tabstop>
  <tabstop>uiEnableLocalSSH</tabstop>
  <tabstop>uiEnableRemoteHTTP</tabstop>
  <tabstop>uiEnableRemoteSSH</tabstop>
  <tabstop>uiNetworkInterface</tabstop>
  <tabstop>uiHTTPPort</tabstop>
  <tabstop>uiRandomiseHTTPPort</tabstop>
  <tabstop>uiSSHPort</tabstop>
  <tabstop>uiRandomiseSSHPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
